transmit-lib/src/cmr/transmit/search.clj

```diff
@@ -19,21 +20,22 @@
    config/token-header (config/echo-system-token)))
 
 (defn-timed save-subscription-notification-time
```

Comment:
This function shouldn't have been put in here. This should be put in the `cmr.transmit.metadata-db2` namespace as it is talking to metadata-db app not search app.

Reply:
Also this function has a bad name, it is not just for saving subscription notification time.